app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -31,4 +43,63 @@ class StateFragmentPresenter @Inject constructor(
   fun setAudioFragmentVisible(isVisible: Boolean) {
     getStateViewModel().setAudioFragmentVisible(isVisible)
   }
+
+  private fun showInputInteractions() {
+
```

Comment:
As mentioned earlier, we do not need extra black space after opening and closing bracket of any function.

Reply:
removed blank space